crypto/rand/randfile.c

```diff
@@ -101,6 +100,12 @@ int RAND_load_file(const char *file, long bytes)
         else
             n = RAND_FILE_SIZE;
         i = fread(buf, 1, n, in);
```

Comment:
and does this read a full buffer ?

Reply:
It would depend on the implementation of _fread_ but I'd expect so.  _setvbuf_ to the rescue?